packages/ColorPicker/src/ColorPicker.stories.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
The styling rule is not supposed to be double quotes now?

Reply:
yes it is. For some reason it doesn't correct it for .stories pages :( I can manually fix this

Reply:
I will take a time today to look at the EsLint rules to englobe `.stories.tsx` and `.test.tsx` files.

Reply:
Is it a bug or a feature?
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/4323800/5e551702-f09a-48ff-a46b-3136be93f426)

Reply:
Will had to ask @franckgaudin about this.